documentation/3.4.0/setup/docker/project-with-survey/mac-os/README.md

```
@@ -0,0 +1,230 @@
+<div align="center">
+
+# Projects Service
+
+<a href="https://shikshalokam.org/elevate/">
+<img
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)